conduct.md

```diff
@@ -7,16 +7,282 @@
+  * Interrupting a team member with criticism without offering him/her the
+    opportunity to share their views
+  * Passing offensive comments during discussions
+  * Trying to dominate a discussion
+  * Talking about topics entirely irrelevant to the discussions
+
+## Decision
+
+Decisions should be made in the environment that every team member's opinion
+is considered important. They should be made collectively as a team or at
+least 80% of the majority's consensus. All information needed to make a
+decision will be clearly stated to every team member and all concerns should
+be clearly expressed. We will use data, rational knowledge and specific
+experience to make decisions instead of popular opinion. Team members need to
+objectively analyzed the options for the best of the team. Trust and respect
+the decision made in order to move on the project.
+
+* **Unacceptable (U):**
+  * Making decisions without notifying team members.
+  * Question the decision after it is made.
+  * Making a decision based on only popular opinion.
+
+## Postmortems
+
+In the postmortems process of the project we will use proper communication and
+teamwork in order to solve problems and issues. Going through postmortems will
+help us to debug and ensure problems that arise get taken care of smoothly and
+surely.
+
+* **Acceptable Postmortems (C)**
+  * Talk in the first part of lab section to discuss where we are in the project.
+  * Team leaders can lead through this discussion.
+  * Teams will bring up problems and errors that occurred, or what worked well.
+  * After finishing a project teams should be assigned to look after the program.
+  * Work as a team to fix problems.
+  * Use proper communication to talk to one another.
+
+* **Unacceptable (U)**
+  * Do not blame anyone for errors or problems
+  * Do not work alone use your team members as a guide
+  * Do not attempt to change things without talking to others
+
+## Conflicts
+
+In the conflicts section of the project, we deal with conflicts in a
+professional manner by communicating any type of issue/ conflict with any of the
+team leaders, technical leads, or the CEO directly.
+
+* **Acceptable Behavior (C)**
+  * Team Members shall:
+    * Not unduly use their position/ influence in this project for personal
+    benefit or to benefit relatives or close associates.
+    * Report a conflict or potential conflict with either a team leader,
+    technical lead or the CEO directly as soon as it arises or at the nearest
+    availability.
+  * Team Leads shall:
+    * Be open and respect both sides looking for the most beneficial option
+    towards the project.
+    * Make decisions with the best interest of the team as a whole upon proper
+    discussion.
+* **Unacceptable Behavior (U)**
+  * Team Members shall NOT:
+    * Bring about personal interests that will bring about conflicts with
+    those of the team as a whole.
+    * Move forward on a conflict that has not yet been discussed or resolved
```

Comment:
This point is unclear -- what does moving forward mean? Resolving it, or making a decision, or what? Clarification is needed, perhaps a rewrite as well.

Reply:
Fixed in a PR.